src/bp/core/config/botpress.config.ts

```diff
@@ -256,7 +255,7 @@ export type BotpressConfig = {
   autoRevision: boolean
   eventCollector: EventCollectorConfig
   /**
-   * @default { "default": { "type": "basic", "allowSelfSignup": false, "options": { "maxLoginAttempt": 0} }}
+   * @default { "default": { "id": "default", "type": "basic", "allowSelfSignup": false, "options": { "maxLoginAttempt": 0} }}
```

Comment:
Typings were a bit off here, but I wouldn't add the id property since it's not used and would be misleading. The real id is the object key, which is extracted when the config is being read. Maybe the id should be made optional, with a ! added to the places where the strategy id is being used